### ISSUE TITLE
Clarification please

### ISSUE THREAD

**Issue by ruwentos** [rianjs/ical.net#538]
We decided to use this library because it is most recommended in the field of RFC 5545 pattern handling.
Our use case is a calendar with recurring events. We want to know exactly which events are due on a certain day.

I studied the examples out there and implemented a test code with one recurring event. The function ```GetOccurrences``` only returns unexpected results so far. But perhaps this is because of my nonproper use. To check that, I have the following questions:

1. The application must work independent of its location. That means, we use only UTC. Does ical.net work correctly with UTC when I do not provide any time zone?
2.  When I add another event at the exact same time and recurrence as a first one but with a different UID, nothing in the ```GetOccurrences``` result changes. Can I not expect the library to see an event as different from another when only the UID is different?

Thanks for considering my request.

**Comment by kibblewhite** [comment]
I doubt I can help much, but I was having the same issue. Then I set the description field and occurrences started appearing. I now just set the description field to a single space/white-space character (can not be empty or null) and things started working. Give it a go if you haven't already?

If that doesn't work then I'm not sure what else to suggest... I'm sure more information might be useful for others too (i.e. sample code etc) and they might be able to offer more help than I can...

Just to confirm I'm working exclusively with UTC times and every one of my unique identifier (UID) fields are different from each other.
